libs/core/README.md

```diff
@@ -0,0 +1,7 @@
+# core
+
+This library was generated with [Nx](https://nx.dev).
+
+## Running unit tests
+
+Run `nx test core` to execute the unit tests via [Jest](https://jestjs.io).
```

Comment:
can we have a better README file, i.e. https://gist.github.com/DomPizzie/7a5ff55ffa9081f2de27c315f5018afc